### PR TITLE
Add hooks functionality

### PR DESCRIPTION
**Pull request by CollinsTatang**
**This branch required the following:**
- To refactor the Calculator component from a class-based component to a functional component with hooks.

### CODE REVIEW

File: src/component/Calculator.js [hunk 1]
```diff
@@ -67,44 +67,40 @@ const buttons = [
...
-              type="button"
-              onClick={this.onClickHandler}
-            />
-          ))}
-        </div>
+  return (
+    <div className="calculator-board">
+      <div className="output-box">
+        <p>{displayNum || '0'}</p>
+      </div>
```
Comment (henatan99) [inline src/component/Calculator.js:94]: - [x] There are errors on the browser when a number is divided by zero or when an operator is clicked twice. As your app should work without errors please incorporate some error handling for `displayNum`. 
Screenshot from 2021-08-12 22-50-28
Screenshot from 2021-08-12 22-50-18

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @CollinsTatang , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @CollinsTatang, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CollinsTatang** [comment]
Hi @henatan99 and @Dipeshtwis 
Thanks for the review massage.

**Pull request merged**